m4_sessions/1711-inning/Group_8.md

```diff
@@ -1,9 +1,31 @@
-Group Eight
+Group Eight 
 
-Session Title: Description: Why should students come to this session?
+Session Title: Transforming your Narrative through Growth Mindset
 
-Session Outcomes: What will students leave with or gain from this session?
+Description: Why should students come to this session?
+
+Session Outcomes: What will students leave with or gain from this session? 
+
+  Students will be able to:
+    Identify when they are in a fixed mindset.
```

Comment:
Build from the Mod 1 growth mindset session: https://github.com/turingschool/gear-up/blob/master/Growth_Mindset_Facilitator_Guide.markdown